From ac5bf59a55a2c86345966800ae8b4e06dbb48375 Mon Sep 17 00:00:00 2001
From: Jeremi Piotrowski <jpiotrowski@microsoft.com>
Date: Thu, 21 Apr 2022 16:17:07 +0200
Subject: [PATCH] use altRootDir for flatcar as well

---
 dirs/dirs.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dirs/dirs.go b/dirs/dirs.go
index 38a5cdc120..041690a93d 100644
--- a/src/github.com/snapcore/snapd/dirs/dirs.go
+++ b/src/github.com/snapcore/snapd/dirs/dirs.go
@@ -338,6 +338,7 @@ func SetRootDir(rootdir string) {
 		"arch",
 		"archlinux",
 		"fedora",
+		"flatcar",
 		"gentoo",
 		"manjaro",
 		"manjaro-arm",
-- 
2.32.0

